rpcs3/rpcs3qt/gamepads_settings_dialog.cpp

```diff
@@ -16,6 +17,9 @@
 
 input_config input_cfg;
 
+inline std::string sstr(const QString& _in) { return _in.toUtf8().toStdString(); }
```

Comment:
I think I've mentioned it before, but `toUtf8()` is unnecessary since [according to the documentation](http://doc.qt.io/qt-5/qstring.html#toStdString) it calls it during `toStdString` anyway.